sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -242,6 +242,10 @@ private[hive] class SparkExecuteStatementOperation(
       val executionHiveClassLoader = sqlContext.sharedState.jarClassLoader
       Thread.currentThread().setContextClassLoader(executionHiveClassLoader)
 
+      if (!runInBackground) {
+        parentSession.getSessionState.getConf.setClassLoader(executionHiveClassLoader)
```

Comment:
do we need to set it back?

Reply:
> do we need to set it back?

No need, in our env, just changed like this works fine, but maybe set it back is more better to avoid potential problem.

Reply:
"about module"?

Reply:
> "about module"?

emmmm, module about hive..